insights/specs/default.py

```diff
@@ -356,6 +356,7 @@ def semid(broker):
     ls_ocp_cni_openshift_sdn = simple_command("/bin/ls -l /var/lib/cni/networks/openshift-sdn")
     ls_sys_firmware = simple_command("/bin/ls -lanR /sys/firmware")
     ls_var_lib_mongodb = simple_command("/bin/ls -la /var/lib/mongodb")
+    ls_R_var_lib_nova_instances = simple_command("/bin/ls -laR /var/lib/nova/instances")
```

Comment:
This spec also needs to be defined in insights_archive.py.

Reply:
Done. Thanks